docker-compose.yaml

```diff
@@ -287,10 +288,10 @@ services:
       # Self-hosted redis endpoints.
       - YOLOX_GRPC_ENDPOINT=page-elements:8001
       - YOLOX_HTTP_ENDPOINT=http://page-elements:8000/v1/infer
-      - YOLOX_INFER_PROTOCOL=grpc
+      #- YOLOX_INFER_PROTOCOL=grpc
       # build.nvidia.com hosted endpoints.
       #- YOLOX_HTTP_ENDPOINT=https://ai.api.nvidia.com/v1/cv/nvidia/nv-yolox-page-elements-v1
-      #- YOLOX_INFER_PROTOCOL=http
+      - YOLOX_INFER_PROTOCOL=http
```

Comment:
GRPC is not performant today with these YOLOX NIMs, so switched to http, lets switch to grpc once we make those chanes